vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/descriptor/FieldSchema.kt

```diff
@@ -1,10 +1,12 @@
 package org.vitrivr.engine.core.model.descriptor
 
+import org.vitrivr.engine.core.model.types.Type
+
 /**
  * A [FieldSchema] describes a field in a [Descriptor]. This is required for a [Descriptor]'s ability to
```

Comment:
I still think we should try to harmonise our nomenclature, however this is not the priority of this PR.

The schema consists of fields (i.e. the descriptors) which in turn have fields (i.e. attributes).